pages/about.html

```diff
@@ -4,5 +4,32 @@
 layout: about
 home: false
 ---
-
-<h2>About Content Goes Here</h2>
\ No newline at end of file
+<div class="aboutPage">
+  <div class="section">
+    <div class="infoContainer">
+      <h1>Who is Code for America?</h1>
```

Comment:
There are a few of these headings on this page, so let's use an H2 instead

Reply:
so the Trello card instructions say "For the typography use a H1 instead of H2". still want me to change to H2?

Reply:
Ah, yes, there should only ever be one H1 on a page.
Joseph means it should look like the H1 style he has in his theme.
I'll let him know to prevent future confusion.